### PR TITLE
Combined with Francesca's and leave system

### PR DESCRIPTION
**Pull request by LunaLiu829**
Hi all, this is combined with Francesca's and leave system and comment feature added.

### CODE REVIEW

File: roomranger/src/main/java/org/launchcode/roomranger/controllers/ManagerHomepageController.java
Comment (MuntoniFrancesca) [inline roomranger/src/main/java/org/launchcode/roomranger/controllers/ManagerHomepageController.java:1]: I don't understand this part?
Did you add code that you commented out?
Just to understand what happened :D

Comment (LunaLiu829) [reply roomranger/src/main/java/org/launchcode/roomranger/controllers/ManagerHomepageController.java:1]: No, just my old one have your templates I think, and I merged with your in main....

Comment (MuntoniFrancesca) [reply roomranger/src/main/java/org/launchcode/roomranger/controllers/ManagerHomepageController.java:1]: Ok!

**Comment by MuntoniFrancesca** [comment]
Looks good!

**Review by MuntoniFrancesca — APPROVED** [review]
Looks good!

**Review by Sonal2808 — APPROVED** [review]

**Review by sammu1991 — APPROVED** [review]
Good to go!

**Pull request merged**